### PR TITLE
Update some stuff to lodash

### PR DESCRIPTION
**Pull request by Somotaw**

### CODE REVIEW

**Comment by TooAngel** [comment]
DCBOT: 
This repository is under [democratic collaboration](https://github.com/TooAngel/democratic-collaboration) and will be merged automatically.

The merge decision is based on the outcome of the reviews:
 - `Approve` add the reviewer value (number of commits) to the `metric`
 - `Request changes` substract the reviewer value from the `metric`

 The merge will happen after `(1 - metric/total.votes) * (5 + pull_request.commits * 5))` calculated from the last code change.

Please review the PR to make a good democratic decision.

Summoning some reviewers:
 - @TooAngel: Who are you?
 - @Nevrdid: I always like your reviews
 - @WhiteHalmos: I just want to get this merged ASAP

File: src/config_brain_stats.js [hunk 1]
```diff
@@ -116,7 +116,8 @@ brain.stats.init = function() {
       room: {}
     }
   };
-  let rolesNames = _(Game.creeps).map(c => c.memory.role).countBy(function(r) { return r; }).value();
+  let rolesNames = _.map(Game.creeps, 'memory.role', _.countBy(function(r) { return r; }));
```
Comment (Nevrdid) [inline src/config_brain_stats.js]: the actual version is a loadash chain so don't really need to change it :) and it's surely slower.

Comment (Somotaw) [inline src/config_brain_stats.js]: but you run .map in it, not _.map

Comment (Somotaw) [inline src/config_brain_stats.js]: or am i missing something?

Comment (Nevrdid) [inline src/config_brain_stats.js]: _(...).map.anotherLodashFct().anotherOne...value()

**Comment by TooAngel** [comment]
DCBOT: A new review, yeah.

    Votes: 127/296
    Coefficient: 0.429054054054
    Merging in 5 days 12 hours
    Age 0 days 4 hours

**Comment by TooAngel** [comment]
DCBOT: A new review, yeah.

    Votes: 225/296
    Coefficient: 0.760135135135
    Merging in 2 days 5 hours
    Age 0 days 4 hours

**Pull request merged**